File: market-application/src/main/java/com/example/market/application/command/BuyNowCommand.java
package com.example.market.application.command;

import com.example.market.domain.catalog.SkuId;
import com.example.market.domain.shared.UserId;

public record BuyNowCommand(
        String idempotencyKey,
        UserId buyerId,
        SkuId skuId
) {}

File: market-application/src/main/kotlin/com/example/market/application/command/BuyNowCommand.kt
package com.example.market.application.command

import com.example.market.domain.catalog.SkuId
import com.example.market.domain.shared.UserId

@JvmRecord
data class BuyNowCommand(
    val idempotencyKey: String,
    val buyerId: UserId,
    val skuId: SkuId,
)
